OpenDreamRuntime/Procs/Native/DreamProcNativeRoot.cs

```diff
@@ -1854,7 +1854,9 @@ public static DreamValue NativeProc_splittext(DreamObject instance, DreamObject
             DreamList list = DreamList.Create();
 
             foreach (DreamValue type in arguments.GetAllArguments()) {
-                DreamPath typePath = type.GetValueAsPath();
+                if (!type.TryGetValueAsPath(out var typePath)) {
```

Comment:
This proc also accepts DreamObjects

Reply:
This proc apparently also takes strings like `"/obj"`. DreamLists need to be ignored too.